graphs/cmumps-patient.fbp

```diff
@@ -0,0 +1,31 @@
+# cmumps-patient.fbp
+
+# Takes a patient ID and returns a string.
```

Comment:
It depends on what you query with the HTTP request to a REST end-point (usually Conor's for us, but it could be anything from a REST endpoint with an ID).  

This is a modified version of James's original cmumps-patient-jsonld.fbp graph.  His version is calling parse-json directly after getting the REST response.  I don't want to do that given our renaming changes since I have to manipulate the response first before I can call Mike's translation code - it will fail if I do not.  This is one of the issues I want to talk with you about.